xen/arch/arm/coproc/plat/coproc_xxx.c

```diff
@@ -90,8 +89,27 @@ static int vcoproc_xxx_write(struct vcpu *v, mmio_info_t *info, register_t r,
     dev_dbg(coproc_xxx->dev, "write r%d=%"PRIregister" offset %#08x base %#08x\n",
             dabt.reg, r, offset, (uint32_t)mmio->addr);
 
+#if 1
     /* for debug purposes */
```

Comment:
As I understand all the code here is for debug, so why do you stress it here and wrap with "#if 1"?